entrypoint.sh

```diff
@@ -37,20 +41,7 @@ git config --global --add safe.directory $GITHUB_WORKSPACE
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-ktlint_version=$(ktlint --version)
-echo "ktlint version: $ktlint_version"
-
-if [ "$(printf '%s\n' "0.49.0" "$ktlint_version" | sort -V | head -n1)" = "0.49.0" ]; then
-  if [ "$INPUT_ANDROID" = true ]; then
-    export ANDROID="--code-style=android_studio"
-  else
-    export ANDROID="--code-style=intellij_idea"
-  fi
-else
-  if [ "$INPUT_ANDROID" = true ]; then
-    export ANDROID=--android
-  fi
-fi
```

Comment:
On one hand `--code-style=android_studio` is what ktlint is saying to use with ktlint 0.49.0 but apparently this paramter has no effect (neither on CI, nor locally)

On the other hand `--android` is throwing error
```
ERROR com.pinterest.ktlint.cli.internal.KtlintCommandLine - Option '--android' / '-a' is deprecated and replaced with option '--code-style=android_studio'. Setting '.editorconfig' property 'ktlint_code_style=android_studio' might be a better idea for a project that is always to formatted with this code style.
```
but still seems to work just fine. 

If this `--code-style` parameter really does not work with ktlint 0.49.0 (which was released few days ago) I would expect 0.49.1 to appear soon, meaning it would make more sense probably to keep `action-ktlint` as is hoping that problems will be solved soon on ktlint side

Reply:
I raised an issue https://github.com/pinterest/ktlint/issues/1982
Let's see if it can get fixed soon.

Reply:
Issue will be fixed in next ktlint version. I will close the PR and raise another one to apply these changes starting v0.49.1